torchvision/datasets/usps.py

```diff
@@ -75,9 +75,10 @@ def __getitem__(self, index: int) -> Tuple[Any, Any]:
         """
         img, target = self.data[index], int(self.targets[index])
 
-        # doing this so that it is consistent with all other datasets
-        # to return a PIL Image
-        img = Image.fromarray(img, mode='L')
+        if self.get_image_backend() == 'PIL':
+            # doing this so that it is consistent with all other datasets
+            # to return a PIL Image
+            img = Image.fromarray(img, mode='L')
```

Comment:
Does this mean that `img` is in `HWC` format? If that's the case, we would want to make it `CHW` in the `torch` backend, for consistency with how the rest of pytorch (and torchvision now) handles images.

Additionally, isn't `img` a numpy `ndarray`? We would need to convert it to a torch tensor via `torch.as_tensor(img)` in order to have a tensor.

Reply:
MNIST is already Tensor in itself I think, I haven't checked the other datasets. Will check them all.